test/functional/helpers.lua

```diff
@@ -430,6 +466,20 @@ local function create_callindex(func)
   return table
 end
 
+-- Helper to skip tests. Returns true in Windows systems.
+-- pending_func is the pending() from busted
+local function pending_win32(pending_func)
+  clear()
+  if os_name() == 'windows' then
```

Comment:
I think it would be equivalent. I don't expect the performance to differ, since the value is cached in both cases.